docker-compose.yml

```diff
@@ -42,3 +42,5 @@ services:
 
   broker:
     image: redis:4.0.9
+    depends_on: 
+        - db
```

Comment:
I'd rather remove this change from this particular PR because:

1. This has nothing to do with the PR title
2. Actually this is something undocumented (and a PR or an issue documenting it would be useful), but I rather **not** include `db` in _any_ `depends_on` so this gives me the freedom to use a remote database for data persistence and, later, to be accessed in a web dashboard; thus spinning up the `db` via Docker is purposely optional and manual

Reply:
Regarding this case, let me know what you think of 3265b3ee9f71c18c6a8f840b33e6c8e687a40d6a ; )

Reply:
Sorry, my bad! I'm new to docker-compose, when I saw that db not up, I think that forgot to put on depends_on, because broker exit when db it's off!